backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/75-de.metas.fresh.reports/5489110_sys_gh3701_fix_missing_product_cat_fresh_ADR_umsatzliste_bpartner_report_function.sql

```diff
@@ -1,87 +1,90 @@
 ﻿
 DROP FUNCTION IF EXISTS report.fresh_ADR_umsatzliste_bpartner_report
-	(
-		IN Base_Period_Start date,
-		IN Base_Period_End date, 
-	
-		IN issotrx character varying,
-		IN C_BPartner_ID numeric, 
-		
-		IN M_AttributeSetInstance_ID numeric,
-		
-		IN AD_Language character varying
-	);
-	
+(
+    IN Base_Period_Start date,
+    IN Base_Period_End date,
+
+    IN issotrx character varying,
+    IN C_BPartner_ID numeric,
+
+    IN M_AttributeSetInstance_ID numeric,
+
+    IN AD_Language character varying
+);
+
 DROP FUNCTION IF EXISTS report.fresh_ADR_umsatzliste_bpartner_report
-	(
-		IN Base_Period_Start date,
-		IN Base_Period_End date, 
-	
-		IN issotrx character varying,
-		IN C_BPartner_ID numeric, 
-		
-		IN M_AttributeSetInstance_ID numeric,
-		
-		IN AD_Language character varying,
-		IN AD_Org_ID numeric
-	);
+(
+    IN Base_Period_Start date,
+    IN Base_Period_End date,
+
+    IN issotrx character varying,
+    IN C_BPartner_ID numeric,
+
+    IN M_AttributeSetInstance_ID numeric,
+
+    IN AD_Language character varying,
+    IN AD_Org_ID numeric
+);
 DROP TABLE IF EXISTS report.fresh_ADR_umsatzliste_bpartner_report;
 
```

Comment:
These changes shall appear in DDL folder as well; We need to maintain the DDLs.
So please, update DDL for `fresh_ADR_umsatzliste_bpartner_report `and `fresh_ADR_umsatzliste_bpartner_report`